src/postgres/commands/createPostgresServer/steps/PostgresServerCreateStep.ts

```diff
@@ -6,6 +6,7 @@
 import PostgreSQLManagementClient from 'azure-arm-postgresql';
 import { Progress } from 'vscode';
 import { AzureWizardExecuteStep, callWithMaskHandling, createAzureClient } from 'vscode-azureextensionui';
+import { postgresDefaultStorageSizeMB } from '../../../../constants';
```

Comment:
Move the constant into this file, close to where it's actually used. We don't want to use the "constants.ts" file for every single constant we have otherwise it'll get out of hand quickly